sbin/build.sh

```diff
@@ -473,12 +475,12 @@ createOpenJDKTarArchive()
         zip -r -q OpenJRE.zip ./"${JRE_TARGET_PATH}"
       EXT=".zip" ;;
       aix)
-        GZIP=-9 tar -cf - ./"${OPENJDK_REPO_TAG}"/ | gzip -c > OpenJDK.tar.gz
-        GZIP=-9 tar -cf - ./"${JRE_TARGET_PATH}"/ | gzip -c > OpenJRE.tar.gz
+        GZIP=-9 tar -cf - ./"${OPENJDK_REPO_TAG}"/ | $COMPRESS -c > OpenJDK.tar.gz
+        GZIP=-9 tar -cf - ./"${JRE_TARGET_PATH}"/  | $COMPRESS -c > OpenJRE.tar.gz
```

Comment:
whitespace again

Reply:
In this case I just lined it up with the line above to make it look better but happy to change it back